### DISCUSSION TITLE
2 Sites Schema

### DISCUSSION THREAD

**Discussion by hjonathan2019** [coinfoundry/miningcore#1476] [General]
Hello, im running now one instance of node and miningcore.
I want to add another site, and a masterminingcore to increase and secure uptime of my mining pool

is this schema correct?? 

how can i do it? 

![image](https://user-images.githubusercontent.com/59586038/196231627-feefe0bb-47d0-4e1d-bf08-3a986029b4bc.png)

**Comment by seperatis** [comment]
This is very easy. You just need to run a web server on server B, and on server A.
Put the HTML files in /var/www/html and connect the websites to your miningcore public API.

As far as running 3 nodes, you need to use shareRelay. It's not hard, but not documented extremely well. search closed issues for shareRelay and you will find enough info to get it done (that's how I got it done).

That's basically it.
